superset-frontend/src/components/ReportModal/index.tsx

```diff
@@ -307,6 +283,13 @@ const ReportModal: FunctionComponent<ReportProps> = ({
     </>
   );
 
+  const renderErrorMessage = () => {
+    if (currentReport?.error) {
+      return errorMapping[currentReport.error] || currentReport?.error;
+    }
+    return currentReport?.error;
```

Comment:
if 287 is false, would this also be falsy/null/undefined? Should we just return nothing?